src/components/Camera/index.js

```diff
@@ -31,7 +32,27 @@ const CaptureActions = ({handeClick}) =>
     </button>
   </div>
 
-const CameraPure = ({method, autoCapture, onUploadFallback, onFallbackClick, onUserMedia, faceCaptureClick, countDownRef, webcamRef, onWebcamError}) => (
+type CameraCommonType = {
```

Comment:
In which cases we need to specify the type? Does it need to be done every time that a constant is defined?

Reply:
typing is optional, however for class React components (not pure components), Flow mandates prop types. There is also type inference done by Flow, so even without specifying stuff you can get some type checking for free, but it doesn't seem to work super well for react components.